libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -1159,26 +1159,34 @@ void IRGeneratorForStatements::appendExternalFunctionCall(
 			if iszero(extcodesize(<address>)) { revert(0, 0) }
 		</checkExistence>
 
-		let <pos> := <freeMem>
+		let <pos> := mload(<freeMemoryPointer>)
+
 		mstore(<pos>, <shl28>(<funId>))
 		let <end> := <encodeArgs>(add(<pos>, 4) <argumentString>)
 
-		let <result> := <call>(<gas>, <address>, <value>, <pos>, sub(<end>, <pos>), <pos>, <retSize>)
-		if iszero(<result>) { <forwardingRevert> }
+		let <result> := <call>(<gas>, <address> <value>, <pos>, sub(<end>, <pos>), <pos>,
```

Comment:
Why no `,` between address and value?

Reply:
because that value is optional, so if it is needed, <value> evaluates to ", VALUE" and if not, it evaluates to an empty string - in order to avoid double comma in the param-list.

Reply:
Oh I see! Maybe `<?hasValue>, <value> </hasValue>`?